### ISSUE TITLE
SetCookie per request

### ISSUE THREAD

**Issue by janekolszak** [go-resty/resty#257]
Hi,
Shouldn't there be a SetCookie method for a Request?

**Comment by jeevatkm** [comment]
@janekolszak yeah currently no easy to set cookie method at request each level. As you know it only exists at the client level.

I will add it, it's fairly straight forward to implement. If you're interested you can take a stab at it.

Stopgap suggestion:

Maybe you can try using the [SetPreRequestHook](https://godoc.org/github.com/go-resty/resty#Client.SetPreRequestHook) until this enhancements come.

**Comment by janekolszak** [comment]
I'm just using SetHeader for now

**Comment by jeevatkm** [comment]
@janekolszak I have added cookie method for request instance in branch `add-set-cookie-in-request`. Can you try it out and let me know?
